scripts/core/interactive-article-actions-panel/utils/get-initial-destination.ts

```diff
@@ -18,11 +18,11 @@ export function getInitialDestination(
     const currentDeskId = sdApi.desks.getCurrentDeskId();
 
     let destinationDesk: string = (() => {
-        if (lastDestination?.type === 'desk' && lastDestination.desk != null) {
-            return lastDestination.desk;
-        } else if (currentDeskId != null) {
+        if (currentDeskId != null) {
             return currentDeskId;
-        } else if (items.length === 1 && items[0].task?.desk != null) {
+        }else if (lastDestination?.type === 'desk' && lastDestination.desk != null) {
+            return lastDestination.desk;
+        }else if (items.length === 1 && items[0].task?.desk != null) {
```

Comment:
Did you test the other use cases and make sure that these changes don't break anything else?

Reply:
we've updated the functionality based on angular logic, thus when we open a **sendto and publish tab**, we receive the `currentDesk` instead of the `lastDestinationDesk`.

Reply:
our `otherDeskSelected `logic fails due to this behaviour. now its works fine as expected.